internal/config/validator.go

```diff
@@ -58,13 +58,69 @@ func (v *validator) Validate(resources ...client.ObjectList) error {
 			clusterResources.Nodes = append(clusterResources.Nodes, list.Items...)
 		}
 	}
+	clusterResources = resetfields(clusterResources)
 	_, err := For(clusterResources, v.validate)
 	if errors.As(err, &TransientError{}) { // we do not want to make assumption on ordering in webhooks.
 		return nil
 	}
 	return err
 }
 
+func resetfields(clusterResources ClusterResources) ClusterResources {
```

Comment:
resetfields is a bit too generic. Please add a proper name and a comment that says what are the fields that we are resetting.

Reply:
I agree , I changed the name to be `resetTransientErrorsFields`.